tools/utils.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
Please place to https://github.com/open-mmlab/mmdetection/blob/master/mmdet/utils/misc.py